components/encoder/fake/encoder.go

```diff
@@ -69,7 +71,8 @@ func (e *fakeEncoder) Reconfigure(
 
 // Config describes the configuration of a fake encoder.
 type Config struct {
-	UpdateRate int64 `json:"update_rate_msec,omitempty"`
+	UpdateRate int64   `json:"update_rate_msec,omitempty"`
+	Speed      float64 `json:"speed,omitempty"`
```

Comment:
Should this have units? If so, what would they be?

I was thinking about:
```suggestion
	Speed      float64 `json:"speed_tpm,omitempty"`
```
where tpm means ticks per minute, but I'm not sure if that is clear? (or because it is fake, if `speed` is fine)

Reply:
maybe just `ticks_per_sec`, since that kinda matches the [guidelines](https://viam.atlassian.net/wiki/spaces/ENG/pages/262371/Viam+Engineering+Guidelines#Resource-API-Naming-Conventions)

Reply:
`tpm` wouldn't be obvious to me. I think because it's fake, `speed` is fine, in the way that, when you're playing a board game, the units on the currency are often just called "money" ("I will pay 3 money to buy that potion").

Reply:
Oh, I like John's suggestion! `ticks_per_sec` or `ticks_per_min` is clearer than `tps` or `tpm`. My main problem is that `ticks` is such a rare unit that I couldn't tell what the `t` stood for.

Reply:
Oh, I really like `ticks_per_sec` @JohnN193 . That makes sense to me. Will swap to that (might involve a few other code changes because the rest of this logic is measured in ticks per minute)